docs/AppFramework.md

```diff
@@ -0,0 +1,320 @@
+* `storageType` describes the type of remote storage. Currently `s3` i:w
+  s the only supported type
+* `provider` describes the remote storage provider. Currently `aws` & `minio` are the supported providers 
+* `endpoint` helps configure the URI/URL of the remote storage endpoint that hosts the apps
+* `secretRef` refers to the K8s secret object containing the remote storage access key
+* `path` describes the path (including the bucket) of one or more app sources on the remote store 
+
+### appSources
+
+`appSources` helps configure the name & scope of the appSource, as well as remote store's volume & location
+
+* `name` uniquely identifies the App source configuration within a CR. This is to locally used by the Operator to identify the App source
+* `scope` defines the scope of the App to be installed. 
+  * If the scope is `local` the apps will be installed locally on the pod referred to by the CR
+  * If the scope is `cluster` the apps will be installed across the cluster referred to by the CR
+  * The cluster scope is only supported on CR's that manage cluster-wide app deployment
+  
+    | CRD Type          | Scope support  | App Framework support |
+    | :---------------- | :------------- | :-------------------- |
+    | ClusterManager    | cluster, local | Yes                   |
+    | SearchHeadCluster | cluster, local | Yes                   |
+    | Standalone        | local          | Yes                   |
+    | LicenceMaster     | local          | Yes                   |
+    | IndexerCluster    | N/A            | No                    |
+
+* `volume` refers to the remote storage volume name configured under the `volumes` stanza (see previous section)
+* `location` helps configure the specific appSource present under the `path` within the `volume`, containing the apps to be installed  
+
+### appsRepoPollIntervalSeconds
+
+`appsRepoPollIntervalSeconds` helps configure the polling interval(in seconds) to detect addition or modification of apps on the Remote Storage
+
+
+
+## App Framework Limitations
+
+The App Framework does not review, preview, analyze, or enable Splunk Apps and Add-ons. The administrator is responsible for previewing the app or add-on contents, verifying the app is enabled, and that the app is supported with the version of Splunk Enterprise used in the containers. For App packaging specifications see [Package apps for Splunk Cloud or Splunk Enterprise](https://dev.splunk.com/enterprise/docs/releaseapps/packageapps/) in the Splunk Enterprise Developer documentation. The app archive files must end with .spl or .tgz; all other files are ignored.
+
+1. The App Framework has no support to remove an app or add-on once it’s been deployed. To disable an app, update the archive contents located in the App Source, and set the app.conf state to disabled. If the container instance is running Splunk Web, you can also use the Apps UI to disable an app on an individual container.
```

Comment:
Can we please refrain from recommending UI for making changes to App?

Reply:
My bad, we already removed it, but missed it when i am combining them.